build.gradle

```
@@ -143,6 +138,7 @@ subprojects {
 	repositories {
 		mavenCentral()
 		mavenLocal()
+		maven { url = uri("http://nexus.phx.connexta.com:8081/nexus/content/groups/public/") }
```

✏️ Nit: This limits the scanning to Connexta artifacts.
```suggestion
		maven { 
		     url = uri("http://nexus.phx.connexta.com:8081/nexus/content/groups/public/")
		     content {
		            includeGroup "com.connexta"
		      } 
		}
```